applications/nrf_desktop/description.rst

```diff
@@ -327,7 +327,7 @@ Depending on the development kit you use, you need to select the respective conf
 
       .. table-from-rows:: /includes/sample_board_rows.txt
          :header: heading
-         :rows: nrf52840gmouse_nrf52840
+         :rows: nrf52840gmouse/nrf52840
```

Comment:
this is wrong, have you tried building docs?

Reply:
Sorry, didn't know that. I've dropped this change.